stylesheet/main.css

```diff
@@ -151,6 +158,10 @@ nav{
     
     
 }
-ANCHOR{
-    border-bottom:3.5em;
+.back-to-top:hover {
+    opacity: 1;
+}

```

Comment:
This rule causes the opacity to change when the back-to-top link is hovered over